docs/resources/integration.md

```
@@ -10,6 +10,24 @@ description: |-
+## Example Usage
+
+```terraform
+# Create a monitor first
+resource "uptimerobot_monitor" "website" {
+  name     = "My Website"
+  type     = "http"
+  url      = "https://example.com"
+  interval = 300
+}
+
+resource "uptimerobot_integration" "slack" {
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)